notebooks/02_analyses/sentiment_analysis.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>can you add <code>plt.show()</code> here to prevent  the <code>&lt;AxesSubplot:xlabel='sentiment', ylabel='count'&gt;</code>  output. same for below image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>